x/mint/client/cli/cli_test.go

```diff
@@ -0,0 +1,28 @@
+// +build cli_test
+
+package cli_test
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/tests/cli"
+	"github.com/cosmos/cosmos-sdk/x/mint/client/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCLIMintQueries(t *testing.T) {
+	t.Parallel()
+	f := cli.InitFixtures(t)
+
+	proc := f.SDStart()
+	t.Cleanup(func() { proc.Stop(false) })
+
+	params := testutil.QueryMintingParams(f)
+	require.NotNil(t, params)
```

Comment:
What does `QueryMintingParams`? Struct ot pointer to struct? If it's a struct, it'd better to check it against its zero value (empty structy)